lib/utils.ts

```diff
@@ -228,7 +228,7 @@ export function parseOutput(
 }
 
 export function parseRustOutput(lines: string, inputFilename?: string, pathPrefix?: string) {
-    const re = /^ --> <source>[(:](\d+)(:?,?(\d+):?)?[):]*\s*(.*)/;
+    const re = /^ +--> <source>[(:](\d+)(:?,?(\d+):?)?[):]*\s*(.*)/;
```

Comment:
Maybe `\s+` would be more appropriate in general? (and for the other space here)?

Reply:
I think it’s unlikely that `rustc` would ever emit a tab or a Unicode space here, but it probably doesn’t hurt either.

Reply:
Thanks; it's more for the look of the thing when the rest of it uses `\s` for whitespace (even though I bet it doesn't use tabs there either)